include/libcyphal/presentation/presentation.hpp

```diff
@@ -74,7 +136,17 @@ class Presentation final : private detail::IPresentationDelegate
         return Publisher<Message>{publisher_impl};
     }
 
-    /// @brief Makes a message subscriber.
+    /// @brief Makes a typed publisher bound to its standard (aka fixed) subject id.
```

Comment:
```suggestion
    /// @brief Makes a typed publisher bound to its fixed subject id.
```

Please do not call it standard. The correct term is fixed. There are similar comments below.